sidebars.js

```diff
@@ -61,6 +46,41 @@ const sidebars = {
         id: "how-to/index",
       },
       items: [
+        "how-to/run-dev-env",
+        "getting-started/deploy",
+        "getting-started/sync-with-github",
+        "getting-started/sync-with-bitbucket",
+        "getting-started/sync-with-git-differences-between-plans",
+        {
+          type: "category",
+          label: "Customize Your Service",
+          link: {
+            type: "doc",
+            id: "custom-code/index",
+          },
+          items: [
+            "custom-code/managing-custom-files",
+            "custom-code/add-business-logic",
+            "custom-code/add-action-to-controller",
+            "custom-code/add-graphql-query",
+            "custom-code/seed-db",
+            "custom-code/add-custom-dto",
+          ],
+        },
+        {
+          type: "category",
+          label: "Run Amplication Locally",
+          link: {
+            type: "doc",
+            id: "running-amplication-platform/connect-server-to-github",
+          },
+          items: [
+            "running-amplication-platform/connect-server-to-github",
+            "running-amplication-platform/connect-server-to-bitbucket",
+            "running-amplication-platform/configure-github-auth",
+            "getting-started/cli"
```

Comment:
Hey @mulygottlieb, I added the CLI page in here with other pages related to running amplication locally.